lib/types/cap.ex

```diff
@@ -43,15 +42,8 @@ defmodule Kadena.Types.Cap do
   defp validate_name(name) when is_binary(name), do: {:ok, name}
   defp validate_name(_name), do: {:error, [name: :invalid]}
 
-  @spec validate_args(args :: list()) :: validation()
-  defp validate_args(args) when is_list(args) do
-    case PactValuesList.new(args) do
-      %PactValuesList{} = pact_values -> {:ok, pact_values}
-      {:error, [{_arg, reason}]} -> {:error, [args: reason]}
-    end
-  end
-
-  defp validate_args(%PactValuesList{} = pact_values), do: {:ok, pact_values}
-
+  @spec validate_args(args :: pact_values()) :: validation()
+  defp validate_args([]), do: {:ok, []}
+  defp validate_args([%PactValue{} | _rest] = pact_values), do: {:ok, pact_values}
```

Comment:
@EdwinGuayacan I think this affects the user experience when creating a `Cap`.

How about receiving the list, and internally converting it to a list of `PactValue`'s?

Following the same approach I think it can solve the suggestion that @L-Zuluaga mentions... What do you think?

Reply:
I think it's correct